curation-migration-datasync/eventConsumer.ts

```diff
@@ -55,3 +57,38 @@ export async function removeScheduledItem(
   // Remove association from DynamoDB
   await curatedItemModel.deleteByCuratedRecId(curatedRecord.curatedRecId);
 }
+
+/**
+ * Update the scheduled item in the legacy database based on the curatedRecId
+ * from the DynamoDB. If the DynamoDB mapping is not found, treat the event
+ * the same as an add-scheduled-item event.
+ * @param eventBody
+ * @param db
+ */
+export async function updateScheduledItem(
+  eventBody: ScheduledItemPayload,
+  db: Knex
+) {
+  const curatedItemModel = new CuratedItemRecordModel();
+  const scheduledItem = await curatedItemModel.getByScheduledItemExternalId(
+    eventBody.scheduledItemExternalId
+  );
+
+  if (!scheduledItem) {
+    const errorMessage = `update-scheduled-item: No mapping found for scheduledItemExternalId=${eventBody.scheduledItemExternalId}`;
+    Sentry.captureMessage(errorMessage);
+    console.log(errorMessage);
+    await addScheduledItem(eventBody, db);
+    return;
+  }
+
+  const dbService = new DataService(db);
+  const parserResponse = await getParserMetadata(eventBody.url);
+
+  await dbService.updateScheduledItem(
+    eventBody,
+    scheduledItem.curatedRecId,
+    parseInt(parserResponse.resolvedId),
+    parserResponse.domainId
+  );
+}
```

Comment:
we have to update the dynamo record's lastUpdatedAt.

i have something like this:
 ```
curatedItem.lastUpdatedAt = Math.round(new Date().getTime() / 1000);
await curatedItemModel.insert(curatedItem);
```